Move service list to separate section (#84)
working-group-two/docs.wgtwo.com: docs/provision-api-reference.md

  * **name:** Service name from the service list.
* **userid:** Unique user ID for this user.

**Error cases:**
* msisdn unknown
* service name unknown

> ### Service list

Reviewer: Should this be h3 (is it a child of `## UPDATE`? Or h2?
```suggestion
## Service list
```

Author: It actually is a child of update, as that's the only call in the provisioning API v1 that uses this list.

> The list of services available to provision is found here: [list of services](https://docs.wgtwo.com/subscription-profile/subscription-services/)

Reviewer: Use paths without domain, for when we change the domain name (soon).

```suggestion
The list of services available to provision is found here: [list of services](/subscription-profile/subscription-services/)
```

Reviewer: Unsure if you can use relative path, e.g.:
```suggestion
The list of services available to provision is found here: [list of services](./provision-service-list.md)
```

If that works, that would be my preferred solution.

Author: Your second (preferred) option gave the markdown straight up so it wasn't very pretty, at least locally, but your first suggestion worked fine, so I'll happily grab that! Thank you :pray:
